lib/Attr.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+const Node = require("./Node");
+
+module.exports = class Attr extends Node {
+  constructor(document, elm, name) {
+    super(document, elm.$elm);
+
+    this.elm = elm;
```

Comment:
is `elm` really a property of Attr? You could make a `const kElm = Symbol.for("element");` and set `this[kElm] = elm;`